pkg/svc/zserver.go

```diff
@@ -2,51 +2,142 @@ package svc
 
 import (
 	"context"
+	"flag"
+	"time"
 
 	"github.com/golang/protobuf/ptypes/empty"
 	"github.com/my-app/pkg/pb"
+	rpb "github.com/my-app/pkg/resp_pb"
```

Comment:
I guess, we may have a troubles with non-existing path. app renaming should be done carefully